src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { PersonLink } from '../PersonLink';
+import cn from 'classnames';
+
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(false);
+
+  const { slug } = useParams();
+  const selectedPerson = slug;
+
+  useEffect(() => {
+    setLoading(true);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => setError(true))
+      .finally(() => setLoading(false));
+  }, []);
+
+  return (
+    <main className="section">
+      <div className="container">
+        <h1 className="title">People Page</h1>
+
+        <div className="block">
+          <div className="box table-container">
+            {loading && <Loader />}
+
+            {error && (
+              <p data-cy="peopleLoadingError" className="has-text-danger">
+                Something went wrong
+              </p>
+            )}
+
+            {people.length === 0 && !loading && !error && (
```

Comment:
It will be better to move it into a variable

```suggestion
            const example = !people.length && !loading && !error;
            
            {example && (
```